io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl

```diff
@@ -0,0 +1,411 @@
+   samm:preferredName "Percentage of Material Weight"@en ;
+   samm:description "Share of the full material weight in percentage."@en ;
+   samm:dataType xsd:float ;
+   samm-c:unit :PercentageOfMaterialWeightUnit .
+
+:PercentageConstraint a samm-c:RangeConstraint ;
+   samm:preferredName "Percentage Constraint"@en ;
+   samm:description "Ensure that no percentage values is below zero or above 100."@en ;
+   samm-c:minValue "0.0"^^xsd:float ;
+   samm-c:maxValue "100.0"^^xsd:float ;
+   samm-c:lowerBoundDefinition samm-c:AT_LEAST ;
+   samm-c:upperBoundDefinition samm-c:AT_MOST .
+
+:PreConsumerEntity a samm:Entity ;
+   samm:preferredName "Pre Consumer Entity"@en ;
+   samm:description "Amount of material that is Pre-consumer material. Material diverted from the waste stream during a manufacturing process. Excluded is reutilization of materials such as rework, regrind or scrap generated in a process and capable of being reclaimed within the same process that generated it. Pre-consumer material can be used in form of recovered or recycled material as a substitute for primary material."@en ;
+   samm:properties ( :mechanicalRecycling ) .
+
+:PostConsumerEntity a samm:Entity ;
+   samm:preferredName "Post Consumer Entity"@en ;
+   samm:description "Amount of material that is Post-consumer material. Material generated by households or by commercial, industrial and institutional facilities in their role as end-users of the product which can no longer be used for ist intended purpose. This includes returns of material from the distribution chain. Post-consumer material can be used in form of recovered or recycled material as a substitute for primary material."@en ;
+   samm:properties ( :mechanicalRecycling :chemicalRecycling ) .
+
+:PostConsumerAutomotiveEntity a samm:Entity ;
+   samm:preferredName "Post Consumer Automotive Entity"@en ;
+   samm:description "Material generated by households or by commercial, industrial and institutional facilities in their role as end-users of END OF LIFE VEHICLES which can no longer be used for its intended purpose. This includes returns of material from the distribution chain.\n\nThis attribute ONLY contains post consume material which originates from the automotive industry. All other material must be included in \"post consumer\"."@en ;
+   samm:properties ( :chemicalRecycling :mechanicalRecycling ) .
+
+:PercentageOfMaterialWeightUnit a samm:Unit ;
+   samm:preferredName "Percentage of Material Weight Unit"@en ;
+   samm:referenceUnit unit:percent ;
+   samm:symbol "%" .
+
+:mechanicalRecycling a samm:Property ;
+   samm:preferredName "Mechanical Recycling"@en ;
+   samm:description "Materials that have undergone mechanical recycling."@en ;
+   samm:characteristic :MechanicalRecycling .
+
+:chemicalRecycling a samm:Property ;
+   samm:preferredName "Chemical Recycling"@en ;
+   samm:description "Materials that have undergone chemical recycling."@en ;
+   samm:characteristic :ChemicalRecycling .
```

Comment:
Property and the referenced Characteristic should not have the same name. please check other places also.